src/netstandard/ref/System.Buffers.cs

```diff
@@ -106,7 +121,6 @@ public enum OperationStatus
         public partial struct Enumerator
         {
             private object _dummy;
-            private int _dummyPrimitive;
```

Comment:
Not to nit pick, but why is this getting removed?  The Enumerator does contain value type fields.

Reply:
I'm matching the annotation from [CoreFX](https://github.com/dotnet/corefx/blob/master/src/System.Memory/ref/System.Memory.cs#L390).

Reply:
Do we know why we lack that field in corefx?  I thought if a struct contained at least one value type field, we needed the ref version to also have a value type field; is that not the case?

Reply:
Not sure @ericstj, AFAIK you generated those at some point, right?

Reply:
The field was deleted in CoreFX in nullable annotations PR: https://github.com/dotnet/corefx/pull/38299/files#diff-577bd970484f00965a5e67de411cab76L232 . Looks like a mistake to me.

Reply:
Removed here: https://github.com/dotnet/corefx/commit/ad961d0dc9d9054942c6a7e3a8c96c8684dd16cc#diff-577bd970484f00965a5e67de411cab76L232

@safern was this intentional?

Reply:
This was definitely a mistake on my side. It was not intentional. Investigating why GenAPI removed it when I ran it for my PR. I will however, submit a PR in corefx to fix.

Reply:
Once you have it, we should take it to ship room and align it between .NET Standard and .NET Core.

Reply:
Filed https://github.com/dotnet/standard/issues/1512 for tracking.

Reply:
Ok, so confirmed with @gafter and we indeed need both, the reference type and the value type primitive, because of: https://github.com/dotnet/roslyn/issues/30194

However, given the type we're speaking of, I don't think we should fix it in 3.0 at this point. It seems strange that people would do:

```cs
ReadOnlySequence<int>.Enumerator e;

e.Current;
```

directly, I would think them doing .GetEnumerator() or looping directly in a foreach loop.

Reply:
Also I opened: https://github.com/dotnet/arcade/issues/3891 which is the real reason why this was deleted.

Reply:
We've used the same tool to generate refs for other APIs, old and new. Seems likely the same issue exists in other places, potentially more important?

Reply:
> Seems likely the same issue exists in other places, potentially more important?

Yes, seems likely. I think doing a sanity check is worth it. Will open an issue in corefx and self-assign.